de.fabrik19.mos.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: de.fabrik19.mos.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mos-launcher
+modules:
+  - name: mos-launcher
+    buildsystem: simple
+    build-options:
+      strip: false
+      no-debuginfo: true
+    build-commands:
+      - mkdir -p /app/bin
+      - cp libHarfBuzzSharp.so /app/bin/libHarfBuzzSharp.so
+      - cp libSkiaSharp.so /app/bin/libSkiaSharp.so
+      - cp mos-launcher /app/bin/mos-launcher
+      - cp mos-launcher.pdb /app/bin/mos-launcher.pdb
+      - cp mos-launcher.xml /app/bin/mos-launcher.xml
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - cp icon.svg /app/share/icons/hicolor/scalable/apps/de.fabrik19.mos.Launcher.svg
+      - mkdir -p /app/share/applications/
+      - cp de.fabrik19.mos.Launcher.desktop /app/share/applications/de.fabrik19.mos.Launcher.desktop
+      - mkdir -p /app/share/metainfo/
+      - cp de.fabrik19.mos.Launcher.metainfo.xml /app/share/metainfo/de.fabrik19.mos.Launcher.metainfo.xml
+    sources:
+      - type: archive
+        url: https://download.mobilitysuite.de/launcher/linux-x64-2.0.7.tar.gz
+        sha256: 5764fa40a10b8d9e99aa3926ec48a0e58d662fff9a9c06053cba288a5c547f96
+      - type: file
+        path: icon.svg
+      - type: file
+        path: de.fabrik19.mos.Launcher.desktop
+      - type: file
+        path: de.fabrik19.mos.Launcher.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --filesystem=home:rw
```

Comment:
this is a broad permission to "save settings".

Reply:
It is not just to save setting but also to select a directoy in the home directory which contains a configuration. This configuration is used by the server.